html/reaction.html

```diff
@@ -100,34 +100,34 @@
             type <div class="reaction_identifier_type selector" data-proto="ReactionIdentifier_IdentifierType"></div>
             value <div class="reaction_identifier_value edittext longtext"></div>
             <span class="fa fa-upload text_upload" aria-hidden="true" style="cursor:pointer;" onclick="ord.reaction.setTextFromFile($(this).closest('.reaction_identifier'), 'reaction_identifier_value');"></span>
+            <br>details <div class="reaction_identifier_details edittext"></div>
             <div class="remove" onclick="ord.reaction.removeSlowly(this, '.reaction_identifier');">remove</div>
-            <br>
-            details <div class="reaction_identifier_details edittext"></div>
           </div>
         </div>
         <div onclick="ord.identifiers.add();" class="add">+ add identifier</div>
       </fieldset>
 
-      <fieldset id="section_inputs" class="section">
+      <fieldset id="section_inputs" class="section s2">
         <legend>
           <span class="collapse"></span>
-          Inputs
+          <span class="h2">Inputs</span>
           <span data-toggle="tooltip" data-placement="right" title="Reaction inputs include every chemical added to the reaction vessel.">
             <span class="fa fa-question-circle" aria-hidden="true"></span>
           </span>
         </legend>
           <div id="inputs">
 
             <div id="input_template" class="section input" style="display: none;">
-              <fieldset>
+              <fieldset class="s3">
                 <legend>
                   <span class="collapse"></span>
-                  Input
-                  <span class="input_name_label">name: </span><div class="input_name edittext"></div>
+                  <span class="h3">Input</span>
+                  <div onclick="ord.inputs.remove(this);" class="remove">remove</div>
```

Comment:
I'm confused. Isn't this a new click handler?

Reply:
The diff is lying to you: line 126 and 130 (in the original) are swapped. It's not a new handler.